supereeg/model.py

```diff
@@ -606,7 +614,7 @@ def _bo2model(bo, locs, width=20):
     n, d = _blur_corrmat(sub_corrmat_z, sub_rbf_weights)
     return n, d, 1
 
-def _mo2model(mo, locs, width=20):
+def _mo2model(mo, locs, width=20, disable_parallelization=False):
```

Comment:
(same as above) -- do we still need the `disable_parallelization` flag?

Reply:
nope. removed